controllers/api/file/linkAction/AddFileLinkAction.inc.php

```diff
@@ -109,7 +109,7 @@ function _getTextLabels($fileStage) {
 			),
 			SUBMISSION_FILE_PROOF => array(
 				'wizardTitle' => 'submission.upload.proof',
-				'buttonLabel' => 'submission.addFile'
+				'buttonLabel' => 'submission.changeFile'
```

Comment:
When do submission files during the proof stage appear other than the one grid? If this is too general, probably this whole PR is not appropriate since it uses the file stage throughout.